lxd/daemon.go

```diff
@@ -798,7 +798,7 @@ func (d *Daemon) Init() error {
 	/* Read the uid/gid allocation */
 	d.IdmapSet, err = shared.DefaultIdmapSet()
 	if err != nil {
-		shared.LogWarn("Error reading default idmap", log.Ctx{"err": err.Error()})
+		shared.LogWarn("Error reading default uid/gid map", log.Ctx{"err": err.Error()})
```

Comment:
This is really not important and you can leave it as is but I really feel that whenever we can avoid using `log.Ctx{}` we should do so as order in the log output is not guaranteed and it mostly looks messy in comparison to a nicely formatted `shared.LogWarnf()` or similar. :)